.env.example

```diff
@@ -31,6 +31,13 @@ SERVER_URL=
 ## A string or array used for signing cookies (optional)
 # COOKIE_SECRET=
 
+## https://en.gravatar.com/site/implement/images#rating
+## g: suitable for display on all websites with any audience type.
+## pg: may contain rude gestures, provocatively dressed individuals, the lesser swear words, or mild violence.
+## r: may contain such things as harsh profanity, intense violence, nudity, or hard drug use.
+## x: may contain hardcore sexual imagery or extremely disturbing violence.
+# RATING=g
```

Comment:
Can this be changed to `GRAVATAR_IMAGE_RATING` and moved to the Gravatar section? I don't forsee this variable being used for anything else.